packages/x-tailwindcss/src/x-tailwind-plugin/components/tag/solid.ts

```diff
@@ -0,0 +1,39 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the `solid` styles for component `tag`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+export function tagSolid(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    solid: {
+      backgroundColor: `var(--tag-color-25, ${theme('colors.neutral.10')})`,
+      borderWidth: 0,
```

Comment:
In the designs, it has borderColor and borderWidth

![image](https://user-images.githubusercontent.com/6597815/224261204-fd75dda5-d9f3-42c3-b0f0-a06b344b8ed3.png)

Reply:
You're right! I've aded it.